README.md

```diff
@@ -166,14 +166,14 @@ To create a pre-release or releasing in a different channel:
 
 1. Create a new branch for the channel you want to release to (`alpha`, `beta`, and more).
 2. Create a PR for your feature branch with the channel branch as a base.
-3. Pick a version following this format: `version-channel`. For example, `0.10.9-alpha`, `1-beta`, and more.
+3. Pick a version following this format: `<version>-<channel>`. For example, `0.10.9-alpha`, `1-beta`, and more.
 4. Update the `version` field in `package.json`.
-5. Once you've received at least one approval, merge the Pull Request **with the rebase and merge strategy**.
+5. Once you've received at least one approval, merge the Pull Request **with the "Create a merge commit" strategy**.
 6. Create a [GitHub Release](https://github.com/DataDog/datadog-ci/releases/new?target=alpha&tag=0.10.9-alpha&prerelease=1&title=Alpha+prerelease):
    - Target the channel branch.
-   - Pick a tag based on your version `version-channel`.
+   - Pick a tag based on your version `<version>-<channel>`.
    - Check the `This is a pre-release` checkbox.
-7. Publish the release and an action publishes it on npm.
+7. Publish the release and an action will publish it on npm.
```

Comment:
```suggestion
7. Publish the release and an action publishes it on npm.
```

Reply:
Ah, thank you! Didn't about this rule at all 😁